setup.py

```diff
@@ -1,15 +1,22 @@
 from distutils.command.build import build
+import re
```

Comment:
nit:  code quality `import blah` should go above the `from x import y` with \n in between the two sets of imports.